Library/Homebrew/brew.sh

```diff
@@ -427,14 +436,16 @@ Your curl executable: $(type -p $HOMEBREW_CURL)"
 Minimum required version: ${HOMEBREW_MINIMUM_GIT_VERSION}
 Your Git version: $major.$minor.$micro.$build
 Your Git executable: $(unset git && type -p $HOMEBREW_GIT)"
-    if [[ -z $HOMEBREW_GIT_PATH || -z $HOMEBREW_DEVELOPER ]]; then
+
+    if [[ -n $HOMEBREW_GIT_PATH || -z $HOMEBREW_DEVELOPER ]]
```

Comment:
```suggestion
    if [[ -n $HOMEBREW_GIT_PATH ]]
```

I think a manually specified `git` path should always fail if it's too old.